service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -130,6 +130,15 @@ public String getUserEmailFromSam(AuthenticatedUserRequest userRequest)
     }
   }
 
+  public String getProxyGroupEmail(String userEmail) throws InterruptedException {
```

Comment:
nit: add javadoc for public methods
Your note about this method in the PR description is helpful, could you also include it in javadoc here?